--- compute-classpath.sh	2014-03-26 15:18:12.378678407 -0400
+++ compute-classpath.sh.alexie	2014-03-26 15:41:40.311890606 -0400
@@ -64,6 +64,11 @@
   CLASSPATH="$CLASSPATH:$FWDIR/streaming/target/scala-$SCALA_VERSION/test-classes"
 fi
 
+if [ -d "$HADOOP_HOME/share/hadoop/mapreduce/lib/" ] ; then
+  HADOOP_MR_LZO_JAR=`ls $HADOOP_HOME/share/hadoop/mapreduce/lib/*lzo*.jar`
+  CLASSPATH=$CLASSPATH:$HADOOP_MR_LZO_JAR
+fi
+
 # Add hadoop conf dir if given -- otherwise FileSystem.*, etc fail !
 # Note, this assumes that there is either a HADOOP_CONF_DIR or YARN_CONF_DIR which hosts
 # the configurtion files.
