--- SparkBuild.scala	2014-03-26 15:18:12.478678913 -0400
+++ altiscale.SparkBuild.scala	2014-03-26 15:22:28.917976156 -0400
@@ -28,13 +28,13 @@
   // Hadoop version to build against. For example, "1.0.4" for Apache releases, or
   // "2.0.0-mr1-cdh4.2.0" for Cloudera Hadoop. Note that these variables can be set
   // through the environment variables SPARK_HADOOP_VERSION and SPARK_YARN.
-  val DEFAULT_HADOOP_VERSION = "1.0.4"
+  val DEFAULT_HADOOP_VERSION = "2.2.0"
 
   // Whether the Hadoop version to build against is 2.2.x, or a variant of it. This can be set
   // through the SPARK_IS_NEW_HADOOP environment variable.
   val DEFAULT_IS_NEW_HADOOP = false
 
-  val DEFAULT_YARN = false
+  val DEFAULT_YARN = true
 
   // HBase version; set as appropriate.
   val HBASE_VERSION = "0.94.6"
